backend/tests/v5/e2e/routes/teamspaces/projects/models/common/tickets.test.js

```diff
@@ -1277,6 +1297,115 @@ const testUpdateManyTickets = () => {
 	});
 };
 
+const testGetTicketHistory = () => {
+	describe('Get ticket history', () => {
+		const { users, teamspace, project, con, fed } = generateBasicData();
+		const template = ServiceHelper.generateTemplate();
+
+		const conTicket = ServiceHelper.generateTicket(template);
+		const fedTicket = ServiceHelper.generateTicket(template);
+
+		const conTicketUpdate = ServiceHelper.generateTicket(template);
+		const fedTicketUpdate = ServiceHelper.generateTicket(template);
+
+		const ticketUpdate = { title: ServiceHelper.generateRandomString() };
+		const updateDate = new Date();
+
+		beforeAll(async () => {
+			await setupBasicData(users, teamspace, project, [con, fed],
+				[template]);
+			await ServiceHelper.db.createTicket(teamspace, project, con, conTicket);
+			await ServiceHelper.db.createTicket(teamspace, project, fed, fedTicket);
+			await ServiceHelper.db.createTicket(teamspace, project, con, conTicketUpdate);
+			await ServiceHelper.db.createTicket(teamspace, project, fed, fedTicketUpdate);
+
+			// Simulate ticket update and log entry for conTicket
+			await insertOne(teamspace, 'tickets.logs', {
```

Comment:
put this in a function and call it with params to reduce duplication with the one below